functional-tests/infinitest-runner-spock-test/src/test/groovy/org/infinitest/parser/JavaAssistClassSpockSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package org.infinitest.parser
+
+import javassist.ClassPool
+import javassist.NotFoundException
+import org.infinitest.testdata.HierarchicSpockSpec
+import org.infinitest.testdata.ParameterizedSpockSpec
+import org.infinitest.testdata.SimpleSpockSpec
+import spock.lang.Specification
+
+class JavaAssistClassSpockSpec extends Specification {
+
+    private ClassPool classPool
+
+    void setup() {
+        classPool = new ClassPool()
+        classPool.appendSystemPath()
```

Comment:
It's enough in Idea, but with Maven, the tests fails with:

```
[ERROR]   JavaAssistClassSpockSpec.should detect Spock 2 test (#clazz.simpleName):21 Condition failed with Exception:

classFor(clazz).isATest()
|        |
|        class org.infinitest.testdata.SimpleSpockSpec
java.lang.RuntimeException: javassist.NotFoundException: org.infinitest.testdata.SimpleSpockSpec
	at org.infinitest.parser.JavaAssistClassSpockSpec.classFor(JavaAssistClassSpockSpec.groovy:30)
	at org.infinitest.parser.JavaAssistClassSpockSpec.should detect Spock 2 test (#clazz.simpleName)(JavaAssistClassSpockSpec.groovy:21)
Caused by: javassist.NotFoundException: org.infinitest.testdata.SimpleSpockSpec
	at javassist.ClassPool.get(ClassPool.java:450)
	at org.infinitest.parser.JavaAssistClassSpockSpec.classFor(JavaAssistClassSpockSpec.groovy:28)
	... 1 more
```

I extracted it to a separate module to do not have to put Spock 2 on the class path (and possibly any other testing framework which would start to be supported). Maybe you have an idea for the workaround?